binary_search_tree/BinarySearchTree.py

```diff
@@ -0,0 +1,120 @@
+class Node(object):
+    # Node for Binary search Tree and set the Node
+    def __init__(self, data):
+        self.data = data
+        self.left = None
+        self.right = None
+
+
+class Bst(object):
+    # insert data in tree
+    def insert(node, data):
+        if node is None:
+            # create a new node
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `pep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpfqi8ue1b/binary_search_tree/BinarySearchTree.py
+++ b/tmp/tmpfqi8ue1b/binary_search_tree/BinarySearchTree.py
@@ -47,7 +47,7 @@
         return node
 
     # delete a node
-    def delete(root,data):
+    def delete(root, data):
         if root is None:
             return root
         if data < root.data:
```